cns/logger/cnslogger.go

```diff
@@ -17,6 +19,9 @@ type CNSLogger struct {
 	DisableMetricLogging bool
 	DisableEventLogging  bool
 
+	zapLogger        *zap.Logger
+	enableETWLogging bool
```

Comment:
I don't think the logger needs to carry this config around. Use the config to instantiate the logger, then we simply use the logger.

Reply:
I had this flag to ensure zap logger instance is not nil, as we don't have zap implementation for Linux. With the Nop core I wouldn't need this. I will remove them.